requirements.txt

```diff
@@ -1,34 +1,34 @@
-asgiref==3.2.1
+asgiref==3.2.7
 auth0-python==3.9.1
-boto3==1.9.212
-botocore==1.12.212
-channels==2.2.0
-channels-redis==2.4.0
+boto3==1.12.37
+botocore==1.15.37
+channels==2.4.0
+channels-redis==2.4.2
 Django>=2.2.4
 django-crequest==2018.5.11
 django-dotenv==1.4.2
-django-extensions==2.2.1
+django-extensions==2.2.9
 django-filter==2.2.0
-django-health-check==3.10.2
-djangorestframework==3.10.2
-django-redis==4.10.0
-djproxy==2.3.4
-elasticsearch-dsl==7.0.0
-gunicorn==19.9.0
+django-health-check==3.12.1
+django-redis==4.11.0
+djangorestframework==3.11.0
+djproxy==2.3.5
+elasticsearch-dsl==7.1.0
+gunicorn==20.0.4
 Jinja2>=2.10.1,<3.0
-kubernetes==10.0.1
+kubernetes==11.0.0
```

Comment:
Looks like we should stick with `kubernetes==10.x`  because of a breaking change in their API.

I poke this to try and use `kubernetes==10.1.0` which should be compatible (at least by the look of the semantic version) but it seems like upgrading this requires an older version of `PyYAML` which is puzzling, see below:

```python
kubernetes 10.1.0 has requirement pyyaml~=3.12, but you'll have pyyaml 5.1.2 which is incompatible.
```

I say it's puzzling because `kubernetes==10.0.1` was happy with `PyYAML==5.1.2` but `kubernetes==10.1.0` seems to want an older version, weird 🤔 (`pyyaml~=3.12`)

Reply:
Seem to have fixed the failing test ☺️

Reply:
You :star:

Reply:
It sounds like the breaking change is a simple rename:

> kubernetes.client.apis package is renamed to kubernetes.client.api

From: https://github.com/kubernetes-client/python/blob/master/CHANGELOG.md#v1100a1

If that's the case, maybe we can simply change the name and use the latest version of `kubernetes` library?

Reply:
I'm :+1: on that suggested change from @davidread. Getting this ticket to a merge-able state is one of today's low-hanging-fruit tasks for me. :kiwi_fruit: :banana: :strawberry: